psychopy/visual/textbox2/textbox2.py

```diff
@@ -769,13 +769,13 @@ def _getStartingVertices(self):
     def draw(self):
         """Draw the text to the back buffer"""
         # Border width
-        self.box.setLineWidth(self.pallette['lineWidth']) # Use 1 as base if border width is none
```

Comment:
Ditto my previous comment - not sure about changing names of actual things used in code

Reply:
I agree. See my initial comment:
> Please also note that the following changes break the API by modifying property names. Not sure how stable that part of the API is supposed to be. If that's a problem, please do tell me, I can revert these specific changes and later propose a more complex change that would preserve backwards compatibility: